server/views/templates/work.config.js

```diff
@@ -0,0 +1,9 @@
+import mockJson from '../../test/mocks/wellcomecollection-api.json';
+
+export const name = 'work';
+export const handle = 'work-template';
+export const status = 'wip';
+
+export const context = {
+  work: Object.assign({}, mockJson, {license: 'CC-BY-NC', imgLink: 'https://wellcomecollection-miro-images.imgix.net/V0047000/V0047696.jpg'}) // temp license and img data until we know how this is coming from the API
+};
```

Comment:
FIY, we might be moving to this model, as it's very nice.
https://wiki.creativecommons.org/wiki/Best_practices_for_attribution#This_is_an_ideal_attribution

Thanks to Hellen and @dannybirchall